back-end/src/base/calls/calls.repository.ts

```diff
@@ -35,4 +38,115 @@ export class CallsRepository {
 		const options = { projection: { _id: 1 } };
 		return this.calls.findOne(query, options);
 	}
+	async updateUserPresence(userId: ObjectId, courseId: string, presence: boolean) {
```

Comment:
For the sake of time, it's alright i suppose.
Even if the goal of a mongodb repository is to be generic and be the entry point of a single table.